### PR TITLE
Update list-keyboard-maestro-macros extension

### PR DESCRIPTION
**Pull request by not-first**
## Description

The extension has been reformatted to use the useCachedPromise hook.
An action on each macro has been added to enable/disable the macro, as well as copy its uid.
A preference option added to show disabled macros, which appear as greyed out subtitles compared to titles.
Fuzzy searching for macro names has been implemented.
Simple README added.

## Screencast

list-keyboard-maestro-macros-1
list-keyboard-maestro-macros-2
list-keyboard-maestro-macros-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by eluce2 — APPROVED** [review]
lgtm

**Comment by pernielsentikaer** [comment]
Can we keep the default values for preferences not to disturb current users too much @not-first

**Comment by not-first** [comment]
No worries. I just reverted the changes back now, so the default values should be the same as they were before I touched the extension. 👍

File: extensions/keyboard-maestro/package.json [hunk 1]
```diff
@@ -6,12 +6,13 @@
...
   "contributors": [
-    "suaw3k"
+    "suaw3k",
+    "fakerr"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "list",
+      "name": "list-macros",
```
Comment (pernielsentikaer) [inline extensions/keyboard-maestro/package.json:15]: Can you use the same name, this is the unique key 🙂

Comment (not-first) [reply extensions/keyboard-maestro/package.json:15]: No worries, I just swapped it back.

**Comment by pernielsentikaer** [comment]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**